.github/workflows/azure-static-web-apps-gentle-beach-0eb2d061e.yml

```diff
@@ -7,6 +7,9 @@ on:
     paths:

```

Comment:
This wasnt loading anything but a black screen when I tried your link! Just a heads up.

Reply:
Updated, I could see it now :)

Reply:
I broke things while trying to set a background picture. It's fine